src/vega.schema.ts

```diff
@@ -50,6 +50,10 @@ export type VgScale = {
   zero?: boolean
 }
 
+export function isVgData(data: string | VgData): data is VgData {
```

Comment:
This method returns true if the variable you pass into it is a `VgData` and not a string. The nice thing is that you can use it as a type guard (hence the `data is VgData`). See how it's being used.